src/erlcloud_aws.erl

```diff
@@ -194,15 +202,23 @@ format_timestamp({{Yr, Mo, Da}, {H, M, S}}) ->
 default_config() ->
     case get(aws_config) of
         undefined ->
-            #aws_config{access_key_id=os:getenv("AWS_ACCESS_KEY_ID"),
-                        secret_access_key=os:getenv("AWS_SECRET_ACCESS_KEY")};
+            AccessKeyId = case os:getenv("AWS_ACCESS_KEY_ID") of
+                              false -> "";
```

Comment:
agree. I'm not sure there was specific reason for that. Anyway, tests are fine with that both in your repo and ours, so I've changed this to be undefined